recipe_templates/mpdaf/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
```

Comment:
Ok, it works so let's go with it. I pushed a fix to remove the version-dependent url, then it should be fine.